src/Core/Domain/ApiAccess/Exception/ApiAccessConstraintException.php

```diff
@@ -45,7 +45,8 @@ class ApiAccessConstraintException extends ApiAccessException
     public const DESCRIPTION_TOO_LARGE = 10;
     public const INVALID_SCOPES = 11;
     public const NON_INSTALLED_SCOPES = 12;
-    public const INVALID_SECRET = 13;
+    public const NOT_POSITIVE_LIFETIME = 13;
```

Comment:
```suggestion
    public const NEGATIVE_LIFETIME = 13;
```
It could have been simpler no? 😅 But just a detail